org.gnome.Atomix.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: org.gnome.Atomix
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: atomix
+rename-icon: atomix
+rename-desktop-file: atomix.desktop
+rename-appdata-file: atomix.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=session-bus
```

Comment:
no. you should determine which names are needed.

run with `--log-session-bus`.

Reply:
> error: Unknown option --log-session-bus

Reply:
Works with `--own-name=org.gnome.atomix`, and it should tell you that the application id is wrong, and in [fact it is](https://gitlab.gnome.org/GNOME/atomix/-/commit/2b9b719582423251c14d0bfbfa2c81dc26e81b49#c9a4a258f31bf5583d4df244150a62a7e4a0aba0_0_2).

Reply:
Thanks.

Reply:
Next time, use the `--log-session-bus` option to debug dbus user session access. One thing to note is that dbus logging won't work with `--socket=session-bus`, I guess because with this permission we're bypassing dbus-proxy.  
Other useful tools to monitor dbus messages: Bustle, D-Feet, dbus-monitor, and bustcl.